### PR TITLE
Adding Conformance Document for Kubernetes 1.13 release.

### PR DESCRIPTION
**Pull request by brahmaroutu**
The Document is built on release branch release-1.13.

### CODE REVIEW

**Comment by brahmaroutu** [comment]
@spiffxp I am creating the conformance doc. There is still few entries that require some fixes. Some of the release information is empty and hence removed from the file.

File: docs/KubeConformance-1.13.md [hunk 1]
```diff
@@ -0,0 +1,1538 @@
...
+## [Replication Controller, release pods](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/rc.go#L77)
+
+### Release v1.13
+A Replication Controller (RC) is created, and its Pods are created. When the labels on one of the Pods change to no longer match the RC's label selector, the RC MUST release the Pod and update the Pod's owner references.
+
+
+
+## [Replica Set, run basic image](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apps/replica_set.go#L90)
+
+### Release v1.9
```
Comment (luxas) [inline docs/KubeConformance-1.13.md]: Why does this say v1.9? Does it mean it was added in 1.9 or?

File: docs/KubeConformance-1.13.md [hunk 2]
```diff
@@ -0,0 +1,1538 @@
...
+Note this test needs to be fixed to also test for stderr
+
+Notational Conventions when documenting the tests with the key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+Note: Please see the Summary at the end of this document to find the number of tests documented for conformance.
+
+## **List of Tests**
+
+
+## [Custom Resource Definition, create](https://github.com/kubernetes/kubernetes/tree/release-1.13/test/e2e/apimachinery/custom_resource_definition.go#L41)
```
Comment (luxas) [inline docs/KubeConformance-1.13.md]: Would it be possible to extract the owner of the e2e test from the [sig-foo] e2e test tag?
this e.g. belongs to apimachinery. Going forward we might support multiple owners as well

File: docs/KubeConformance-1.13.md [hunk 3]
```diff
@@ -0,0 +1,1538 @@
+# Kubernetes Conformance Test Suite - v1.9
```
[the remark this answers is not in the document]
Comment (brahmaroutu) [reply docs/KubeConformance-1.13.md]: Fixed.

File: docs/KubeConformance-1.13.md (hunk 1 above)
Comment (brahmaroutu) [reply docs/KubeConformance-1.13.md]: The version indicates the Kubernetes release in which the test was added.

File: docs/KubeConformance-1.13.md (hunk 2 above)
Comment (brahmaroutu) [reply docs/KubeConformance-1.13.md]: This is an external facing document, I am not sure having such information would be necessary.

File: docs/KubeConformance-1.13.md (hunk 1 above)
Comment (luxas) [reply docs/KubeConformance-1.13.md]: ok

File: docs/KubeConformance-1.13.md (hunk 2 above)
Comment (luxas) [reply docs/KubeConformance-1.13.md]: I think it'd make sense but...

**Comment by timothysc** [comment]
> @timothysc We are in the process of automating the process to include in the main docs. In my opinion we should still keep the docs here until we have stable process. 

I'm going to NAK that statement.  Upstream docs is well defined, we should PR there not here.

**Comment by spiffxp** [comment]
@timothysc This is the place and process we have today.  I agree, it's not ideal.  If you want to change it, let's see the proposal to do so, implement, and migrate.  In the meantime let's use what we have.

**Comment by dankohn** [comment]
It seems that we didn't get consensus on this pull request. I will close in 7 days unless there's activity.

**Pull request closed without merging**